### PR TITLE
docs: Add quickstart guide and point out sample conf

### PR DESCRIPTION
**Pull request by matheusd**
This adds a quickstart guide for knowledgeable individuals that want to
get going faster.

It also adds more a more prominent link to the sample config file and
renames it to sample-dcrlnd.conf so that it matches the actual name used
by the daemon.

### CODE REVIEW

**Comment by isuldor** [comment]
Is `ln newaddress p2pkh` right? I'm just reading the help output and the available address types listed are just `p2wkh` and `np2wkh`

**Comment by matheusd** [comment]
Added a commit to fix the reference.

File: docs/QUICKSTART.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- Use Go >= 1.11
+- Git clone as usual
+- `make install`
+- These should now work:
+  - `dcrlnd --version`
+  - `dcrlncli --version`
+
+# Configuration
+
+Create the config file `~/.dcrlnd/dcrlnd.conf`:
```
Comment (fguisso) [inline docs/QUICKSTART.md]: Maybe the OS differences can be quote here.

Comment (matheusd) [reply docs/QUICKSTART.md]: Added.

**Pull request merged**